pages/signup/signup.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<script src="https://kit.fontawesome.com/bbbaa28fd5.js" crossorigin="anonymous"></script>
+		<link rel="stylesheet" href="https://feather-design.netlify.app/Components/main.css" />
+		<link rel="stylesheet" href="../login/login.css" />
+		<title>Signup</title>
+	</head>
+	<body>
+		<header class="header">
+			<h2>Website</h2>
+			<input type="search" name="search" class="header-search" placeholder="Search..." />
+			<i class="fas fa-bars hamburger-menu"></i>
+			<nav class="nav">
+				<ul class="nav-list">
+					<li>
+						<a href="#"><i class="fas fa-user"></i> Profile</a>
+					</li>
+					<li>
+						<a href="#"><i class="fas fa-heart"></i> Wishlist</a>
+					</li>
+					<li>
+						<a href="#"><i class="fas fa-shopping-bag"></i> Bag</a>
+					</li>
+				</ul>
+			</nav>
+		</header>
+		<main>
+			<div class="login-container">
+				<h3>Signup</h3>
+				<div class="input-container">
+					<label for="email">Email Address *</label>
+					<input type="email" id="email" name="email" placeholder="Enter your email" />
+				</div>
+				<div class="input-container">
+					<label for="password">Password *</label>
+					<input type="text" id="password" name="password" placeholder="Enter your password" />
+				</div>
+				<span>Forgot password?</span>
```

Comment:
You could add a `cursor: pointer;` on this as it is also a link or a clickable item.